assignment2.java

```diff
@@ -42,9 +42,9 @@ public static void main(String[] args)
             opcodeToInstruction.put("11111111111", new Instruction("HALT", 'R'));
 
 
-            HashMap<Integer, String> lineNumberToLabel = new HashMap<Integer, String>();
+            HashSet<Integer> linesWithLabelsHashSet = new HashSet<Integer>();
```

Comment:
Kinda picky, but what if we just call this `lineToHasLabel` or just `linesWithLabels`?